tools/ConsoleTypes.natvis

```diff
@@ -122,6 +114,18 @@
         </Expand>
     </Type>
 
+    <Type Name="TextBuffer">
+        <DisplayString>{{ width={_width}, height={_height} }}</DisplayString>
+        <Expand>
+            <Item Name="_firstRow">_firstRow</Item>
+            <Item Name="scratchpad">*(ROW*)_buffer.__ptr_.__value_</Item>
+            <IndexListItems>
+                <Size>(_commitWatermark - _buffer.__ptr_.__value_ - 1) / _bufferRowStride</Size>
+                <ValueNode>*(ROW*)(_buffer.__ptr_.__value_ + _bufferRowStride * ($i + 1))</ValueNode>

```

Comment:
Even if I add the wil.natvis file to our project I can't get the `get()` function to work as without VS complaining about it having side effects. It works fine in a watch window, so I think this is a problem with a natvis file not being able to reference another file? But why do STL types work then? I don't entirely get it...

In any case, this works. It's ugly but it works.